.github/workflows/deliver.yml

```diff
@@ -17,18 +19,28 @@ jobs:
     steps:
       - name: Checkout the repository
         uses: actions/checkout@v2
-      - name: Set FARMOS_VERSION for push event.
-        if: ${{ !github.event.pull_request }}
-        run: |
-          echo "FARMOS_VERSION=${GITHUB_REF:11}" >> $GITHUB_ENV
-          echo "FARMOS_REPO=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
-      - name: Set FARMOS_VERSION for pull request event.
-        if: ${{ github.event.pull_request }}
+      - name: Set default FARMOS_REPO.
+        run: echo "FARMOS_REPO=${GITHUB_REPOSITORY}" >> $GITHUB_ENV
+      - name: Set FARMOS_VERSION for branch push event.
+        if: github.event_name == 'push' && github.ref_type == 'branch'
+        run: echo "FARMOS_VERSION=${GITHUB_REF:11}" >> $GITHUB_ENV
+      - name: Set FARMOS_VERSION for tag push event.
+        if: github.event_name == 'push' && github.ref_type == 'tag'
+        run: echo "FARMOS_VERSION=${GITHUB_REF:10}" >> $GITHUB_ENV
+      - name: Set FARMOS_VERSION and FARMOS_REPO for pull request event.
+        if: github.event_name == 'pull_request'
```

Comment:
I'm curious if there's an advantage/reason to/for switching to this string comparison vs the `github.event.pull_request` boolean check it's replacing on old line #&#x2060;26?

Reply:
I just did this for consistency. Originally we only had two cases: "is not a pull request" and "is a pull request", so the boolean worked for both.

Now, we've got three: "is a push event of a branch", "is a push event of a tag", and "is a pull request event". I could have left the last one alone, but it felt better to use consistent logic in the conditions.